tools/pangolin/pangolin.xml

```diff
@@ -54,6 +61,7 @@ sed 's/,/\t/g' lineage_report.csv > lineage_report.tsv
             <param name="input1" value="test_seqs.fasta" />
             <section name="additional">
                 <param name="write_tree" value="true" />
+                <param name="include_putative" value="true" />
```

Comment:
Perhaps have a test where `include_putative` is false.

Reply:
Default is set to false. In test one it should be tested with the blank statement as that test is all default settings correct?

Reply:
Ah yeah, didn't see test 1. Should default to false.

Reply:
No worries, I doubled checked my planemo test and serve along with Travis CI tests and they behave/look as expected for command inputs